ignite/engine/__init__.py

```diff
@@ -69,6 +69,15 @@ def supervised_training_step(
     Returns:
         Callable: update function.
 
+    Example::
+
+        model = ...
+        optimizer = ...
+        loss_fn = ...
+
+        update_fn = supervised_training_step(model, optimizer, loss_fn, 'cuda')
+        trainer = Engine(update_fn)
```

Comment:
```suggestion

        from ignite.engine import Engne, supervised_training_step

        model = ...
        optimizer = ...
        loss_fn = ...

        update_fn = supervised_training_step(model, optimizer, loss_fn, 'cuda')
        trainer = Engine(update_fn)
```

Maybe, showing how to import could be helpful.

Reply:
Added in https://github.com/pytorch/ignite/pull/1661/commits/8ad516d07c05fa70d33acf93daeca4bac4181d96